app-backend/tile/src/it/resources/application.conf

```diff
@@ -0,0 +1,18 @@
+gatling {
+  host = "https://app.staging.rasterfoundry.com"
+
+  refreshToken = ""
+  refreshToken = ${?REFRESH_TOKEN}
+
+  projectId = ""
+  projectId = ${?PROJECT_ID}
+
+  randomSeed = 42
+
+  tms {
+    template = "/tiles/${projectId}/$z/$x/$y?token=${authToken}"
```

Comment:
I *think* this should be okay: the refresh token endpoint is hit only once per Gatling run. If it has been overburdened lately, that's due to
1. An earlier design in which I had every gatling "user" hit said endpoint
2. My manually running many gatling runs which each grab the JWT